Content.Server/Lathe/Components/LatheProducingComponent.cs

```diff
@@ -1,24 +1,23 @@
 using Content.Shared.Research.Prototypes;
-using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
 
-namespace Content.Server.Lathe.Components
+namespace Content.Server.Lathe.Components;
+
+/// <summary>
+/// For EntityQuery to keep track of which lathes are producing
+/// </summary>
+[RegisterComponent]
+public sealed class LatheProducingComponent : Component
 {
     /// <summary>
-    /// For EntityQuery to keep track of which lathes are producing
-    /// <summary>
-    [RegisterComponent]
-    public sealed class LatheProducingComponent : Component
-    {
-        /// <summary>
-        /// The recipe the lathe is currently producing
-        /// </summary>
-        [DataField("recipe", required:true, customTypeSerializer:typeof(PrototypeIdSerializer<LatheRecipePrototype>))]
-        public string? Recipe;
+    /// The recipe the lathe is currently producing
+    /// </summary>
+    [ViewVariables]
+    public LatheRecipePrototype? Recipe = null;
 
-        /// <summary>
-        /// Remaining production time, in seconds.
-        /// </summary>
-        [DataField("timeRemaining", required: true)]
-        public float TimeRemaining;
-    }
+    /// <summary>
+    /// Remaining production time, in seconds.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float TimeRemaining;
```

Comment:
After much back and forth you should just use the finish time. It won't handle pausing properly until we get some niceties around it (i.e. timespanserializer + a way to check how long an entity has been paused) but it will save more work later.